src/coreclr/inc/fstream.h

```diff
@@ -40,8 +40,8 @@ class CFileStream : public IStream
         BOOL Close();
 
     private:
-        LONG                                _cRef;
-        HANDLE                              _hFile;
+        LONG   _cRef;
+        FILE*  _fp;
 
 };
```

Comment:
Can this file be deleted and FILE used instead?

Reply:
It implements the COM IStream interface, but not passed over in most usages. Event pipe usage can be replaced with conditional compilation on OS-native handle like AOT.